nersc-instructions

```diff
@@ -0,0 +1,35 @@
+#### this is a general instruction to install and launch pynta on nersc machines ###
```

Comment:
Should this be in with the documentation? Probably on a new page?

Reply:
I am still updating the detail and not completed yet. I just temporarily wrote the note for myself and people who asked me about installing pynta on nersc. should I create a new page?